src/routes/console/project-[project]/storage/bucket-[bucket]/create-file/create.svelte

```diff
@@ -29,7 +29,7 @@
 
     async function create() {
         try {
-            await uploader.uploadFile(
+            uploader.uploadFile(
```

Comment:
The problem with doing this is:

1. the list of files doesn't update after the file is uploaded
2. the success notification is inaccurate

Maybe it would be better to move:

```
            createFile.reset();
            wizard.hide();
```

up before the `await uploadFile()`?

Reply:
We can't do the reset before the upload otherwise there's nothing to upload :sweat_smile: 
This means that the order needs to be the same, and we can't await the upload otherwise if the dev tries to upload something else, there's still going to be the previous file selected

Reply:
I'll ping Carla about the notification message